napari/_vispy/camera.py

```diff
@@ -3,6 +3,15 @@
 
 from napari._vispy.utils.quaternion import quaternion2euler_degrees
 
+# Note: the Vispy axis order is xyz, or horizontal, vertical, depth,
+# while the napari axis order is zyx / plane-row-column, or depth, vertical,
+# horizontal — i.e. it is exactly inverted. This switch happens when data
+# is passed from napari to Vispy, usually with a transposition. In the camera
+# models, this means that the order of these orientations appear in the
+# opposite order to that in napari.components.Camera.
+VISPY_DEFAULT_ORIENTATION_2D = ('right', 'up', 'towards')  # xyz
+VISPY_DEFAULT_ORIENTATION_3D = ('right', 'down', 'away')  # xyz
+
```

Comment:
Why we use different orientations in 2D and 3D, I do not feel like comment is explaining this.

Reply:
We don't use it, Vispy uses it. I don't know why, it's just something I observed.

Reply:
Added a note in the comment.